src/plugins/intel_gpu/src/graph/program.cpp

```diff
@@ -1451,7 +1452,9 @@ void program::set_layout_optimizer_attributes(layout_optimizer& lo) {
             prim.type() != cldnn::broadcast::type_id() &&
             prim.type() != cldnn::quantize::type_id() &&
             prim.type() != cldnn::non_max_suppression::type_id() &&
-            prim.type() != cldnn::roi_align::type_id()) {
+            prim.type() != cldnn::roi_align::type_id() &&
+            prim.type() != cldnn::arg_max_min::type_id() &&
```

Comment:
Does this PR include blocked format support of arg_max_min kernel?

Reply:
ExperimentalDetectronTopKRois consist of two layers on primitives lavel: arg_max  and  exmperimental_detectron_topk_rois
So to support the whole operation we need to provide blocked support in
exmperimental_detectron_topk_rois primitive
and in arg_max primitive for 4d case